utils/custom.js

```diff
@@ -1,4 +1,5 @@
 const {NOTIFICATION_TOPIC_NAME_PREFIX} = require('../helpers/constants');
+const roundingKarmaScore = require('../helpers/roundingKarmaScore');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `roundingKarmaScore` function is imported but there's no check to ensure that it's a valid function before using it. This could lead to runtime errors if the import fails for any reason. It would be better to add a check after importing to ensure that `roundingKarmaScore` is indeed a function.

```diff
+ if (typeof roundingKarmaScore !== 'function') {
+   throw new Error('roundingKarmaScore is not a function');
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
it's existing function